### PR TITLE
Shorten the name of the test jobs in sig-node-cri

### PR DESCRIPTION
**Pull request by yujuhong**
Also add a 1.11 tab for sig-node-cri.

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -5341,17 +5341,22 @@ dashboards:
...
+  - name: containerd-node-conformance
     test_group_name: ci-containerd-node-e2e
-  - name: sig-node-containerd-node-features
+  - name: containerd-node-features
     test_group_name: ci-containerd-node-e2e-features
 
+- name: sig-node-cri-1.11
+  dashboard_tab:
+  - name: docker-node-conformance-1.11
+    test_group_name: ci-kubernetes-node-kubelet-beta
```
Comment (krzyzacy) [inline testgrid/config.yaml]: add the new dashboard to sig-node dashboard groups?

Comment (yujuhong) [reply testgrid/config.yaml]: Oops. Done.

**Pull request merged**